src/views/index.js

```diff
@@ -1,9 +1,9 @@
 import React from "react";
-export default function HomeView (){
+
+export default function HomeView() {
   return (
     <div className="app">
-      <h1>Welcome to Authors haven</h1>
+      <h1>Welcome to Authors haven {process.env.TESTING}</h1>
```

Comment:
Whats the purpose of process.env.TESTING here?

Reply:
@LehruAngela I was using it to test something but I have removed it now.